src/sd.jl

```diff
@@ -41,43 +41,31 @@ function reshape(vectorized_form::Vector{T}, shape::SquareMatrixShape) where T
                         shape.side_dimension)
 end
 
-# Used by the @variable macro. It can also be used with the @constraint macro,
-# this allows to get the constraint reference, e.g.
-# @variable model x[1:2,1:2] Symmetric # x is Symmetric{VariableRef,Matrix{VariableRef}}
-# varpsd = @constraint model x in PSDCone()
+# * If `eltype(Q)` is `AbstractVariableRef`:
```

Comment:
The formatting of this comment makes it a bit hard to follow. If you're writing pseudo-markdown it may be better to use a docstring instead. Same below.